### PR TITLE
Add an API for get action by addres and action type

### PR DESCRIPTION
**Pull request by newyear6565**

### CODE REVIEW

File: queryprotocol/actions/protocol.go [hunk 1]
```diff
@@ -33,27 +33,30 @@ const (
...
-	selectActionCountByType    = selectCount + " WHERE action_type = '%s'"
-	selectXrc20Holders         = "SELECT holder FROM %s WHERE contract='%s' ORDER BY `timestamp` desc limit %d,%d"
-	selectXrc20HistoryByTopics = "SELECT * FROM %s WHERE topics like ? ORDER BY `timestamp` desc limit %d,%d"
-	selectXrcHistoryCount      = selectCount + " WHERE topics like %s"
-	selectXrc20AddressesByPage = "SELECT address, MAX(`timestamp`) AS t FROM %s GROUP BY address ORDER BY t desc limit %d,%d"
-	selectXrc20HistoryByPage   = "SELECT * FROM %s ORDER BY `timestamp` desc limit %d,%d"
-	selectAccountIncome        = "SELECT address,SUM(income) AS balance FROM %s WHERE epoch_number<=%d and address<>'' and address<>'%s' GROUP BY address ORDER BY balance DESC LIMIT %d,%d"
-	selectTotalNumberOfHolders = "SELECT COUNT(DISTINCT address) FROM %s WHERE address<>''"
-	selectTotalAccountSupply   = "SELECT SUM(income) from %s WHERE epoch_number<>0 and address=''"
+	selectEvmTransferCount            = "SELECT COUNT(*) FROM %s WHERE action_type='execution' AND (`from` = '%s' OR `to` = '%s')"
```
Comment (guo) [inline queryprotocol/actions/protocol.go:42]: if this is just indent change, please revert the change to keep the PR clean.

**Comment by ququzone** [comment]
how can we fix CircleCI error?

File: queryprotocol/actions/protocol.go (hunk 1 above)
Comment (newyear6565) [reply queryprotocol/actions/protocol.go]: @guo reverted

File: queryprotocol/actions/protocol.go [hunk 2]
```diff
@@ -134,6 +137,56 @@ func (p *Protocol) GetActionCountByDates(startDate, endDate uint64) (count int,
...
+}
+
+// GetActionsByAddressAndType gets action information list by address and type
+func (p *Protocol) GetActionsByAddressAndType(address, actionType string, offset, size uint64) ([]*ActionInfo, error) {
+	if _, ok := p.indexer.Registry.Find(actions.ProtocolID); !ok {
+		return nil, errors.New("actions protocol is unregistered")
+	}
+	if _, ok := p.indexer.Registry.Find(accounts.ProtocolID); !ok {
+		return nil, errors.New("accounts protocol is unregistered")
+	}
```
Comment (CoderZhi) [inline queryprotocol/actions/protocol.go:153]: ?

File: graphql/resolver.go [hunk 3]
```diff
@@ -684,6 +687,62 @@ func (r *queryResolver) getActionsByType(ctx context.Context, actionResponse *Ac
...
+		offset = 0
+		size = DefaultPageSize
+	case err != nil:
+		return errors.Wrap(err, "failed to get pagination arguments for actions")
+	}
+	count, err := r.AP.GetActionCountByAddressAndType(addr, actionType)
+	if err != nil {
+		return errors.Wrap(err, "failed to get actions' count by address")
+	}
+	actionInfoList, err := r.AP.GetActionsByAddressAndType(addr, actionType, offset, size)
```
Comment (CoderZhi) [inline graphql/resolver.go:718]: if offset >= count, return

File: queryprotocol/actions/protocol.go (hunk 2 above)
Comment (newyear6565) [reply queryprotocol/actions/protocol.go]: @CoderZhi fixed

File: graphql/resolver.go (hunk 3 above)
Comment (newyear6565) [reply graphql/resolver.go:721]: @CoderZhi fixed
